src/Workspaces/VisualBasicTest/Formatting/FormattingTests.vb

```diff
@@ -780,98 +780,139 @@ End Class</Code>
+        Public Async Function LineContinuation3(continuation As String) As Task
+            Dim code = $"Class C
     Sub Method(Optional ByVal i As Integer = 1)
-                    Dim aa = 1 + _
-    2 + _
+                    Dim aa = 1 + {continuation}
+    2 + {continuation}
     3
     End Sub
-End Class</Code>
+End Class"
 
-            Dim expected = <Code>Class C
+            Dim expected = $"Class C
     Sub Method(Optional ByVal i As Integer = 1)
-        Dim aa = 1 + _
-2 + _
+        Dim aa = 1 + {continuation}
+2 + {continuation}
 3
     End Sub
-End Class</Code>
+End Class"
 
-            Await AssertFormatLf2CrLfAsync(code.Value, expected.Value)
+            Await AssertFormatAsync(code, expected)
         End Function
 
-        <Fact, Trait(Traits.Feature, Traits.Features.Formatting)>
-        Public Async Function LineContinuation4() As Task
-            Dim code = <Code>Class C
+        <Theory, Trait(Traits.Feature, Traits.Features.Formatting)>
+        <InlineData("_")>
+        <InlineData("_ ' Comment")>
+        Public Async Function LineContinuation4(continuation As String) As Task
+            Dim code = $"Class C
     Sub Method(Optional ByVal i As Integer = 1)
-                    Dim aa = 1 + _
-            _
-                                          _
-                                     _
-    _
-            _
-    2 + _
+                    Dim aa = 1 + {continuation}
+            {continuation}
+                                          {continuation}
+                                     {continuation}
+    {continuation}
+            {continuation}
+    2 + {continuation}
     3
     End Sub
-End Class</Code>
+End Class"
 
-            Dim expected = <Code>Class C
+            Dim expected = $"Class C
     Sub Method(Optional ByVal i As Integer = 1)
-        Dim aa = 1 + _
- _
- _
- _
- _
- _
-2 + _
+        Dim aa = 1 + {continuation}
+ {continuation}
+ {continuation}
+ {continuation}
+ {continuation}
+ {continuation}
+2 + {continuation}
 3
     End Sub
-End Class</Code>
+End Class"
 
-            Await AssertFormatLf2CrLfAsync(code.Value, expected.Value)
+            Await AssertFormatAsync(code, expected)
+        End Function
+
+        <Theory, Trait(Traits.Feature, Traits.Features.Formatting)>
+        <InlineData("_")>
+        <InlineData("_ ' Comment")>
+        Public Async Function LineContinuation5(continuation As String) As Task
+            Dim code = $"Class C
+    Sub Method(Optional ByVal i As Integer = 1)
+        Dim aa = 1 + {continuation}
+            {continuation}
+                                          {continuation}
+                                     {continuation}
+    {continuation}
+            {continuation}
+    2 + {continuation}
+    3
+    End Sub
+End Class"
+
+            Dim expected = $"Class C
+    Sub Method(Optional ByVal i As Integer = 1)
+        Dim aa = 1 + {continuation}
+ {continuation}
+ {continuation}
+ {continuation}
+ {continuation}
+ {continuation}
```

Comment:
📝 This is the case of particular interest in #38072. Even though none of the other lines changed (indentation was either correct or preserved), these lines that only contained a line continuation moved all the way left.

@paul1956 Is this scenario common enough to _need_ to do anything about it?

Reply:
If people start using this new feature in VB16 then yes, since it is new there is no data. The updated converters do use this to preserve C# comments in VB.

Reply:
@paul1956 It's not super clear to me what the desired behavior here is. Can you file an issue showing the proposed formatting output, so we can compare it to the behavior we see in these tests?

Reply:
@sharwell  I would expect the _ to align to the same place anything that wasn't an _ would. Given below
```
        Sub Method(Optional ByVal i As Integer = 1)
            Dim aa = 1 +
                2 +
                3
        End Sub
```
If you hit enter after the first + before you type anything the cursor is aligned with the 2 if you type anything else except _ it continues to be aligned with the 2. Why should typing an _ change that. If you want I can clone Master and update all the formatting around _ to what I think is desired and do a PR with all the failing tests unless you have another suggestion. The issue I see with the current behavior is it is not just the lines with the _ that shift left but also the following lines. I would be OK if the _ was left justified if everything past it was aligned  the way it would have been have the _ not been present.